.babelrc

```diff
@@ -1,9 +1,10 @@
 {
-  "loose": "all",
-  "modules": "umd",
   "sourceMaps": true,
-  "nonStandard": true,
   "compact": true,
   "comments": false,
-  "jsxPragma": "h"
+  "presets": ["es2015"],
+  "plugins": [
+    "transform-object-rest-spread",
+    ["transform-react-jsx", { "pragma":"h" }]
```

Comment:
I think we'll need to add [transform-es2015-modules-umd](https://babeljs.io/docs/plugins/transform-es2015-modules-umd/#usage-via-babelrc-recommended-) here too.